ui/components/app/add-network/add-network.stories.js

```diff
@@ -10,39 +10,89 @@ export const DefaultStory = () => {
   const MATIC_TOKEN_IMAGE_URL = './images/matic-token.png';
   const ARBITRUM_IMAGE_URL = './images/arbitrum.svg';
   const OPTIMISM_IMAGE_URL = './images/optimism.svg';
+  const AVALANCHE_IMAGE_URL = './images/avax-token.png';
+  const PALM_IMAGE_URL = './images/palm.svg';
+  const BSC_IMAGE_URL = './images/bsc-filled.svg';
 
   const FEATURED_RPCS = [
     {
-      chainId: '0x89',
-      nickname: 'Polygon Mumbai',
-      rpcUrl:
-        'https://polygon-mainnet.infura.io/v3/2b6d4a83d89a438eb1b5d036788ab29c',
-      ticker: 'MATIC',
+      chainId: '42161',
+      nickname: 'Arbitrum One',
+      rpcUrl: 'https://arbitrum-mainnet.infura.io/v3/{INFURA_API_KEY}',
+      ticker: 'AETH',
       rpcPrefs: {
-        blockExplorerUrl: 'https://mumbai.polygonscan.com/',
-        imageUrl: MATIC_TOKEN_IMAGE_URL,
+        blockExplorerUrl: 'https://explorer.arbitrum.io',
+        imageUrl: ARBITRUM_IMAGE_URL,
+      },
+    },
+    {
+      chainId: '43114',
+      nickname: 'Avalanche Mainnet C-Chain',
+      rpcUrl: 'https://api.avax.network/ext/bc/C/rpc',
+      ticker: 'AVAX',
+      rpcPrefs: {
+        blockExplorerUrl: 'https://snowtrace.io/',
+        imageUrl: AVALANCHE_IMAGE_URL,
+      },
+    },
+    {
+      chainId: '56',
+      nickname: 'Binance Smart Chain Mainnet',
```

Comment:
This should be BNB Smart Chain 

https://www.coindesk.com/business/2022/02/15/binance-smart-chain-rebrands-to-bnb-chain/

Reply:
I used a list of the networks provided by the MetaMask team in this Notion file
https://www.notion.so/I-want-to-be-able-to-add-a-popular-custom-network-from-the-custom-network-switcher-modal-7f8c0ae131ea443694396bbf1a36aa1e

Reply:
I commented in that Notion file

Reply:
@amerkadicE please check Barbara's comment on Notion

Reply:
@Gtonizuka I updated network name for the Binance